website/addons/base/testing/logger.py

```diff
@@ -42,19 +30,3 @@ def test_log_deauthorized_when_node_settings_are_deleted(self):
 
         last_log = project.logs[-1]
```

Comment:
This file isn't actually being used anywhere (including when tests are run). I think it was maybe intended to used for generalized mixins like the other files in `/addons/base/testing/`, with the two present tests serving as a reminder of something to test to prevent regressions? Might be way off.

Tests for logging is generally somewhat light, but they should still written. I'll fix up this file and make sure it's actually used.